src/CalculatorFoodItem1/index.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import classNames from 'classnames';
+import RadioButton from '../RadioButton';
+import Spacer from '../Spacer';
+import Typography from '../Typography';
+import Wrapper from '../Wrapper';
+import styles from './styles.css';
+
+function CalculatorFoodItem1(props) {
+  return (
+    <div>
+      <Wrapper
+        className={styles.wrapper}
+        size="standard"
+        >
+        <RadioButton
+          checked={props.checkedEatAmount.toLowerCase() === 'little'}
+          className={classNames({[props.className]: props.className})}
+          label={props.lblEatLittle}
+          onClick={props.onClick}
+          />
+        <Spacer />
+        <RadioButton
+          checked={props.checkedEatAmount.toLowerCase() === 'woman'}
+          className={classNames({[props.className]: props.className})}
+          label={props.lblEatTypicalWoman}
+          onClick={props.onClick}
+          />
+        <Spacer />
+        <RadioButton
+          checked={props.checkedEatAmount.toLowerCase() === 'avg'}
+          className={classNames({[props.className]: props.className})}
+          label={props.lblEatAverage}
+          onClick={props.onClick}
```

Comment:
Every onClick needs to deliver the label to the onClick function, so it knows which label has been clicked :)
Like:
```javascript
onClick={() => props.onClick(props.lblEatAverage)}
```